scanner/updater/manual/data.go

```diff
@@ -0,0 +1,7 @@
+package manual
+
+import (
+	"github.com/quay/claircore"
+)
+
+var manuallyEnrichedVulns = []*claircore.Vulnerability{}
```

Comment:
Nitpick: Add a comment directing people to why this exists on how this is used.

Reply:
> Nitpick: Add a comment directing people to why this exists on how this is used.

It's in the scanner/updater/manual/manual_example_test.go but I can add a comment